### PR TITLE
添加 atsrv_socket 包索引

### PR DESCRIPTION
**Pull request by shaoguoji**
[
1. 添加 atsrv_socket 包索引
]

### CODE REVIEW

File: iot/atsrv_socket/Kconfig [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    config ATSRV_SOCKET_ENABLE_CMD_CIPSEND
+        bool
+        prompt "AT+CIPSEND: send data to network"
+        default y
+        depends on ATSRV_SOCKET_ENABLE_TCPIP
+
+    choice
+        prompt "Version"
+        help
+            Select the cJSON version
```
Comment (BernardXiong) [inline iot/atsrv_socket/Kconfig]: 文本也有问题呢；然后默认版本是哪个？得有个默认版本，并且不能是latest version

**Comment by BernardXiong** [comment]
'''
No.45
atsrv_socket
v1.0.0 : https://github.com/RT-Thread-packages/atsrv_socket/archive/1.0.0.zip
atsrv_socket-1.0.0.zip
Warning : https://github.com/RT-Thread-packages/atsrv_socket/archive/1.0.0.zip is invalid.
The command "cd ${PKGS_ROOT}/packages && python ci.py" failed and exited with 1 during .
'''

需要纠正

File: iot/atsrv_socket/Kconfig (hunk 1 above)
Comment (shaoguoji) [reply iot/atsrv_socket/Kconfig]: 已修改

**Comment by shaoguoji** [comment]
> No.45
> atsrv_socket
> v1.0.0 : https://github.com/RT-Thread-packages/atsrv_socket/archive/1.0.0.zip
> atsrv_socket-1.0.0.zip
> Warning : https://github.com/RT-Thread-packages/atsrv_socket/archive/1.0.0.zip is invalid.
> The command "cd ${PKGS_ROOT}/packages && python ci.py" failed and exited with 1 during .
> 
> 需要纠正

zip 链接无效是因为软件包那边还没有合并，等 release 一版后应该就能用了。

**Comment by shaoguoji** [comment]
@BernardXiong 软件包已经 release，索引测试验证可用。

**Pull request merged**